components/Tabs/index.js

```diff
@@ -7,3 +7,24 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+axios.get('https://lambda-times-backend.herokuapp.com/topics')
+    .then(result => {
```

Comment:
the syntax for your axios request is spot on. You're are making a GET request to the provided URL -- this axios request returns a promise that will either be fulfilled or rejected. 

What I think you need to do is console.log the result to see what it gives you. You have to go digging a little bit to find the information you actually need.  If you DM me on Slack -- @ChristinaK -- I can help you or I can find someone who can help you.